package.json

```diff
@@ -83,6 +83,7 @@
     "eg:goerli:balances": "ts-node -r dotenv/config examples/scripts/balances.ts dotenv_config_path=./examples/scripts/.env.goerli",
     "eg:goerli:export-commitments": "ts-node -r dotenv/config examples/scripts/exportCommitments.ts dotenv_config_path=./examples/scripts/.env.goerli",
     "eg:goerli:import-commitments": "ts-node -r dotenv/config examples/scripts/importCommitments.ts dotenv_config_path=./examples/scripts/.env.goerli",
-    "eg:web-app": "npm run build && cd examples/webApp/ && npm run serve"
+    "eg:web-app": "npm run build && cd examples/webApp/ && npm run serve",
```

Comment:
It is not a mistake at all, but the popular convention within frontend world is to call these script `dev` and `start`. in our case maybe something like `eg:web-app-dev` and `eg:web-app-start`? Just a suggestion of course.

Reply:
nice suggestion, web-app-start sounds nice

Reply:
will drop the react in react-web-app-start after the Vue app is deleted